src/simtools/production_configuration/derive_corsika_limits.py

```diff
@@ -256,6 +267,13 @@ def compute_upper_radius_limit(self, loss_fraction):
             )
             * u.m
         )
+        return self._is_close(
+            radius_limit,
+            self.file_info["core_scatter_max"].to("m")
+            if "core_scatter_max" in self.file_info
+            else None,
+            "Upper radius limit is equal to the maximum core scatter distance of",
```

Comment:
Equal or close? We would log this also if it close but not yet equal (depending on the binning).

Reply:
Technically you are right. But practically, I would consider a core scatter radius of e.g. 10000 m and 10001 m as equal. To you want to me to change it or can we tolerate this slight inconsistency?